### ISSUE TITLE
How to increase scanning area in camera?

### ISSUE THREAD

**Issue by shakato** [SudarAbisheck/ZXing-Orient#10]
im searching for how to increase scanning area in this library, but not find them

**Comment by ivanmagdic** [comment]
Yes, I would like to know too. On devices that have larger screen resolution, scanning area is really small and hard to scan QR/barcodes.

**Comment by chandra2408** [comment]
Have you guys got any solution??

**Comment by nik-sta** [comment]
Hi! Can somebody provide a solution for this topic? @SudarAbisheck maybe you? Cheers and thanks in advance, Nikola

**Comment by nik-sta** [comment]
In my opinion it's also not correct, that the showed scanning area is small and rectangular if I set:

`Barcode.TWO_D_CODE_TYPES`

In my case I just wanna scan QR-Codes and data-matrix codes.

**Comment by haemi** [comment]
any updates on this?

**Comment by kenigj** [comment]
push
